modules/core/client-react-native/App.tsx

```diff
@@ -24,13 +24,15 @@ export default class Main extends React.Component<MainProps> {
         ? `${protocol}//${url.parse(this.props.exp.manifest.bundleUrl).hostname}:${port}${pathname}`
         : __API_URL__;
 
-    const store = createStore(
+    const store = createReduxStore(
       Object.keys(modules.reducers).length > 0
-        ? combineReducers({
+        ? {
             ...modules.reducers
```

Comment:
Why do you create a duplication for `reducers` object?

Reply:
Done